src/server/JasmineGraphServer.cpp

```diff
@@ -1047,87 +999,25 @@ void JasmineGraphServer::copyCentralStoreToAggregateLocation(std::string filePat
+    return Utils::uploadFileToWorker(host, port, dataPort, graphID, filePath, masterIP,
+                                     JasmineGraphInstanceProtocol::UPLOAD_RDF_ATTRIBUTES_CENTRAL);
 }
 
 static bool batchUploadCompositeCentralstoreFile(std::string host, int port, int dataPort, int graphID,
                                                  std::string filePath, std::string masterIP) {
-    return batchUploadCommon(host, port, dataPort, graphID, filePath, masterIP,
-                             JasmineGraphInstanceProtocol::BATCH_UPLOAD_COMPOSITE_CENTRAL);
-}
-
-bool JasmineGraphServer::sendFileThroughService(std::string host, int dataPort, std::string fileName,
-                                                std::string filePath, std::string masterIP) {
-    int sockfd;
-    char data[FED_DATA_LENGTH + 1];
-    socklen_t len;
-    struct sockaddr_in serv_addr;
-    struct hostent *server;
-
-    sockfd = socket(AF_INET, SOCK_STREAM, 0);
-
-    if (sockfd < 0) {
-        server_logger.error("Cannot create socket");
-        return false;
-    }
-
-    server = gethostbyname(host.c_str());
-    if (server == NULL) {
-        server_logger.error("ERROR, no host named " + host);
-        return false;
-    }
-
-    bzero((char *)&serv_addr, sizeof(serv_addr));
-    serv_addr.sin_family = AF_INET;
-    bcopy((char *)server->h_addr, (char *)&serv_addr.sin_addr.s_addr, server->h_length);
-    serv_addr.sin_port = htons(dataPort);
-    if (Utils::connect_wrapper(sockfd, (struct sockaddr *)&serv_addr, sizeof(serv_addr)) < 0) {
-        return false;
-    }
-
-    if (!Utils::sendExpectResponse(sockfd, data, INSTANCE_DATA_LENGTH, fileName,
-                                   JasmineGraphInstanceProtocol::SEND_FILE)) {
-        close(sockfd);
-        return false;
-    }
-
-    server_logger.info("Sending file " + filePath + " through port " + std::to_string(dataPort));
-    FILE *fp = fopen(filePath.c_str(), "r");
-    if (fp == NULL) {
-        close(sockfd);
-        return false;
-    }
-
-    while (true) {
-        unsigned char buff[1024];
-        int nread = fread(buff, 1, sizeof(buff), fp);
-
-        /* If read was success, send data. */
-        if (nread > 0) {
-            write(sockfd, buff, nread);
-        }
-
-        if (nread < sizeof(buff)) {
-            if (feof(fp)) server_logger.info("End of file");
-            if (ferror(fp)) server_logger.error("Error reading file: " + filePath);
-            break;
-        }
-    }
-
-    fclose(fp);
-    close(sockfd);
-    return true;
+    return Utils::uploadFileToWorker(host, port, dataPort, graphID, filePath, masterIP,
+                                     JasmineGraphInstanceProtocol::BATCH_UPLOAD_COMPOSITE_CENTRAL);
 }
 
 static void copyArtifactsToWorkers(const std::string &workerPath, const std::string &artifactLocation,
                                    const std::string &remoteWorker) {
+    // Not used in docker or k8s modes
```

Comment:
It s not clear why you mean by the comment here.

Reply:
This function is never called in docker or k8s modes

Reply:
Ok